app/jobs/importing/run_daily_imports_job.rb

```diff
@@ -249,5 +250,14 @@ def sync_with_cas
       GrdaWarehouse::Tasks::PushClientsToCas.new.sync!
       @notifier.ping('Pushed Clients to CAS')
     end
+
+    def run_external_lookup_exports
+      return unless RailsDrivers.loaded.include?(:hmis_external_apis)
```

Comment:
can we add
`return unless HmisEnforcement.hmis_enabled? && GrdaWarehouse::DataSource.hmis.exists?`
because we don't want this to being until the data source is marked as HMIS